pkg/proxy/backup.go

```diff
@@ -191,7 +191,28 @@ func (p *Proxy) BackupRestoreStatus(ctx context.Context, req *rpc.ProxyEngineReq
 		Status: map[string]*rpc.EngineBackupRestoreStatus{},
 	}
```

Comment:
NIT: same here. don't need to init the response at the beginning.

Reply:
Updated.